test/tests/userobjects/shape_element_user_object/tests

```diff
@@ -39,8 +39,8 @@
   [./shape_side_uo_jac_test]
     type = PetscJacobianTester
     input = 'shape_side_uo_jac_test.i'
-    difference_tol = 1.3e-7
-    ratio_tol = 3e-8
+    difference_tol = 2.4e-7
```

Comment:
What is the reason to change the tolerances? Old PETSc does not work using the existing tolerances?

Reply:
Yes. When I ran the FD Jacobian test with the existing tolerances, the test fails. The existing tolerances look like they're set just above the threshold to allow the FD Jacobian test to pass, so I did the same.

Reply:
This fails with the old PETSc, but works with the current PETSc?

Reply:
The old tolerances of 1.3e-7 and 3e-8 pass with Petsc v3.6.4 but fail with v3.4.5. Increasing these tolerances to 2.4e-7 and 5.4e-8 pass with both versions of Petsc.

Reply:
How about just skip the test for 3.4.5?

Reply:
Sure. I can make those changes.